packages/llvm/llvm.3.9/files/install.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+
+make=$1
+libdir=$2
+
+# Create build dir
+mkdir build
+cd build
+
+# Configure
+cmake ..
+#    -DLLVM_BUILD_LLVM_DYLIB:BOOL=ON \
+#    -DLLVM_LINK_LLVM_DYLIB:BOOL=ON \
+#    ..
+
+# Build
+grep "^ocaml_llvm.*/fast:$" Makefile | tr ":\n" " " | xargs "$make"
```

Comment:
I know, it's depressing :/ I don't know which build-system I find the most depressing, the old one or the new one :/

Anyways, did you take a look at link.patch ? It's really the file that I find the most dubious. I don't know if it works in every case (works for me™). I needed that in order to actually compile/link test code using llvm-3.9. With the old build-system it was handled by a call to llvm-config but now it's all hardcoded :/